tests/Nox.Types.Tests/EntityFrameworkTests/Configuration/StreetAddressDbConfiguration.cs

```diff
@@ -0,0 +1,20 @@
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using Microsoft.EntityFrameworkCore;
+
+namespace Nox.Types.Tests.EntityFrameworkTests;
+
+internal class StreetAddressDbConfiguration : IEntityTypeConfiguration<StreetAddress>
+{
+    public void Configure(EntityTypeBuilder<StreetAddress> builder)
+    {
+        builder.HasKey(e => e.Id);
```

Comment:
this is a compound type, this should be enough:

` builder.OwnsOne(e => e.StreetAddress).Ignore(p => p.Value);`